akka-projection-grpc/src/main/scala/akka/projection/grpc/consumer/scaladsl/GrpcReadJournal.scala

```diff
@@ -477,6 +481,20 @@ final class GrpcReadJournal private (
       }
   }
 
+  // LatestEventTimestampQuery
+  override def latestEventTimestamp(
+      // note that this is actually the producer side defined stream_id
+      // not the normal entity type which is internal to the producing side
+      streamId: String,
+      minSlice: Int,
+      maxSlice: Int): Future[Option[Instant]] = {
+    require(streamId == settings.streamId, s"Stream id mismatch, was [$streamId], expected [${settings.streamId}]")
+    import system.dispatcher
+    addRequestHeaders(client.latestEventTimestamp())
+      .invoke(LatestEventTimestampRequest(streamId, minSlice, maxSlice))
+      .map(_.timestamp.map(_.asJavaInstant))
```

Comment:
not safe to do on parasitic ec?

Reply:
It was same of some other method here. Not performance critical. I guess the conversion from proto timestamp to instant is safe.